src/plugins/notebook/components/Notebook.vue

```diff
@@ -891,13 +916,21 @@ export default {
         },
         startTransaction() {
             if (!this.openmct.objects.isTransactionActive()) {
+                this.activeTransaction = true;
```

Comment:
What happens if there's already an active transaction?

Reply:
There's two situations where we spin up a transaction in notebooks (regarding entries). On a new entry (this will prevent those "blank" entries from popping up randomly when a new entry is created) and when we're editing an existing entry. Once we create a new entry, it's then being edited, this results in a request to start a transaction again, so this is just checking if that's the case, if it is, nothing changes and the transaction will be ended when the entry is blurred.